app/models/sessions/error.rb

```diff
@@ -0,0 +1,13 @@
+class Error
```

Comment:
If you're going to be serializing this class for an API, you might want to consider a different name since Error is generic class in most other languages (javascript, java, etc). You'll have to do custom de-serialization on the clients that consume this